check/doctest

```diff
@@ -7,11 +7,11 @@
 #     https://docs.python.org/3/library/doctest.html
 #
 # Usage:
-#     check/doctest
+#     check/doctest [-q]
```

Comment:
Is it possible to add a comment about what the [-q] flag is doing or what the behavior is if it is enabled?

Reply:
Done.